### DISCUSSION TITLE
Err:access denied error by “user” In heroku

### DISCUSSION THREAD

**Discussion by Chigala** [planetscale/discussion#305] [Bug Reports] [answered]
When I deploy my application to heroku, I get access denied error  but it works just fine in dev. I have added all the env variables I require to heroku, so I don’t seem to know what the problem is

**Comment by fooforge** [comment, marked as answer]
I see that you have also opened a ticket. I've sent a response to the ticket which likely has helped to resolve the issue. If not, please respond to the ticket. Thanks!
